corrector.rb

```diff
@@ -0,0 +1,8 @@
+# Class Corrector file.
+class Corrector
+  def correct_name(name)
+    corrected_name = name.capitalize
+    corrected_name = corrected_name.slice(0, 10) if name.length > 10
+    corrected_name
+  end
```

Comment:
- [ ] Optional suggestion

- Reusing a variable is bad, because of a compiler that would normally warn you about uninitialized variables, etc.

Reply:
Ok, thank's I will improve it! Thank's for the review @Emmyn5600  😄